_config/pom.yml

```diff
@@ -36,6 +37,7 @@ latest:
   documentationHtmlSingle: https://docs.optaplanner.org/7.0.0.CR3/optaplanner-docs/html_single/index.html
   documentationPdf: https://docs.optaplanner.org/7.0.0.CR3/optaplanner-docs/pdf/index.pdf
   javadocs: https://docs.optaplanner.org/7.0.0.CR3/optaplanner-javadoc/index.html
+  kieWorkbenchReleaseNotes: https://docs.drools.org/7.0.0.CR3/drools-docs/html_single/index.html#_wb.releasenotesworkbench.7.0.0.final
```

Comment:
(I 'll handle this PR this week, ignore during your PTO, just mentioning it so you're aware for future)